extras/custom_checks.sh

```diff
@@ -89,12 +89,25 @@ function check_do_not_import_tests_in_hathor() {
 	return 0
 }
 
+function check_do_not_import_from_hathor_in_entrypoints() {
```

Comment:
Why don't we allow `import hathor.cli`? It seems most of the exceptions would fit in this case.

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/887/commits/1ac0f096bb9e39d44dddc541015fc472420a88f5, although I added a more strict version of this, specifically allowing to import `RunNode`.

Reply:
Great! Thanks!